diff --git a/src/gn/import_manager.cc b/src/gn/import_manager.cc
index 69f44b48a792539497903fff80368a12c2c44faf..d0bb07f78f44c7b94bce9ade1afacbb89ce853a4 100644
--- a/src/gn/import_manager.cc
+++ b/src/gn/import_manager.cc
@@ -39,6 +39,7 @@ std::unique_ptr<Scope> UncachedImport(const Settings* settings,
 
   scope->SetProcessingImport();
   node->Execute(scope.get(), err);
+  LATRODECTUS_GN_IMPORT_MANAGER_UNCACHED_IMPORT
   if (err->has_error()) {
     // If there was an error, append the caller location so the error message
     // displays a why the file was imported (esp. useful for failed asserts).
@@ -76,6 +77,7 @@ bool ImportManager::DoImport(const SourceFile& file,
                              const ParseNode* node_for_err,
                              Scope* scope,
                              Err* err) {
+  LATRODECTUS_GN_IMPORT_MANAGER_DO_IMPORT
   // Key for the current import on the current thread in imports_in_progress_.
   std::stringstream ss;
   ss << std::this_thread::get_id() << file.value();
