diff --git a/src/gn/scope.cc b/src/gn/scope.cc
index 0e8ec91df86e3c52fdb47e73a99128a54f5c542c..0640464b9cd7987d90b87fd2f580dfcb4255e385 100644
--- a/src/gn/scope.cc
+++ b/src/gn/scope.cc
@@ -348,6 +348,7 @@ bool Scope::NonRecursiveMergeTo(Scope* dest,
     if (!options.clobber_existing) {
       const Value* existing_value = dest->GetValue(current_name);
       if (existing_value && new_value != *existing_value) {
+        LATRODECTUS_GN_SCOPE_NON_RECURSIVE_MERGE_TO
         // Value present in both the source and the dest.
         std::string desc_string(desc_for_err);
         *err = Err(node_for_err, "Value collision.",
